### ISSUE TITLE
[Bug]: logservice read failed when restart

### ISSUE THREAD

**Issue by jiangxinmeng1** [matrixorigin/matrixone#5669]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 8b742c66b0b3eb on main, or https://github.com/jiangxinmeng1/matrixone/tree/debugmain which add some log
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

logservice return "invalid log range"
<img width="476" alt="1" src="https://user-images.githubusercontent.com/51114574/196071075-9c6d032d-b5a1-4bb9-aa50-666dc7c90b80.PNG">

Lsn 143 is larger than the truncated lsn.
<img width="386" alt="3" src="https://user-images.githubusercontent.com/51114574/196071082-4884f2de-07d2-4422-bdc8-38ad42dda520.PNG">

Before restarted, lsn 143 has been appended succesfully.
<img width="351" alt="4" src="https://user-images.githubusercontent.com/51114574/196071090-03c548e4-8771-47f5-92f7-0c8bf657a3dd.PNG">

### Expected Behavior

read successfully

### Steps to Reproduce

1. Start mo-service.
```Markdown
./mo-service -launch etc/launch-tae-logservice/launch.toml
```
2. Run bvt.
3. Restart.

**Comment by volgariver6** [comment]
resolved in #6012 .

**Comment by aressu1985** [comment]
fixed:1feeb04254c6fdab0ae0061ecc32349136d6ee10

no this panic any more, but still can not restart ,traced by #6284
